src/uniprotkb/components/entry/tabs/genomic-coordinates/styles/entries.module.scss

```diff
@@ -0,0 +1,22 @@
+.coordinates-table {
+  /* | <whitespace> | <no data line> | <whitespace>      | <whitespace> | */
+  /* | <whitespace> | <whitespace>   | <single position> | <whitespace> | */
+  /* | <start>      | <separator>    | <end>             | <whitespace> | */
+  .coordinates {
+    text-align: end;
```

Comment:
The issue here is that if you scroll down to the bottom of the table, because they're all aligned, I'm not too sure how to centre the hyphen more but keep all columns aligned

Reply:
<img width="1372" alt="Screenshot 2024-02-09 at 07 31 55" src="https://github.com/ebi-uniprot/uniprot-website/assets/4357962/ada69226-0591-4285-a989-49c5007004c8">

I hear what you're saying. Looking at non-edge cases (ie not Titin) there still seems to be a little misalignment with the protein coordinates column. Would a bit more horizontal padding help? Or how about having every set of coordinates hug the hyphen eg

```
   12 - 34
  567 - 890
 1234 - 5678
 9012 - 3456
```

Reply:
I have also looked at this quite a bit so perhaps I'm not seeing the forest for the trees and it's actually fine. Maybe good to see what others think about it too.

Reply:
Ah, I see what you mean.
Well, originally we had:
```
     12 - 34
   567 - 890
 1234 - 5678
 9012 - 3456
 ```
 But SIB insisted that we align numbers in order to compare them easily by column:
 ```
   12 -   34
  567 -  890
 1234 - 5678
 9012 - 3456
 ```
 Even though it's true that for big number we might end up in weird-looking edge-cases.
 I'll stick with the SIB requirement for now, as it was explicitely requested like that from them, and we'll see if users complain when/if they see the edge-cases